pinot-spi/src/main/java/org/apache/pinot/spi/data/readers/RecordReaderFileConfig.java

```diff
@@ -54,5 +60,41 @@ public RecordReaderFileConfig(RecordReader recordReader) {
     _dataFile = null;
     _fieldsToRead = null;
     _recordReaderConfig = null;
+    _isDelegateReader = true;
```

Comment:
Could you add comment of what delegate reader (and its ownership) means?

Reply:
makes sense. Explained it clearly.